### PR TITLE
Feature/list of exercises

### PR DESCRIPTION
**Pull request by esrefbozkurt**
US#1
 create components Exercises + Header
 create Header for exercises with muscle group name
 create dummy data
 create a list of exercises by mapping through dummy data
 basic styling components

### CODE REVIEW

File: components/Header/StyledHeader.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import styled from "styled-components";
+
+export const StyledHeader = styled.h1`
+  font-family: Arial, Helvetica, sans-serif;
+  display: flex;
+  justify-content: center;
+  padding: 10px;
+  margin: 0;
```
Comment (ufelgen) [inline components/Header/StyledHeader.js]: not sure if you need to specify "margin: 0" if that's your default setting in the GlobalStyles.js

File: components/Main/StyledExercises.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components";
+
+export const StyledExercises = styled.ul`
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+  margin: 0;
```
Comment (ufelgen) [inline components/Main/StyledExercises.js]: same as comment in Header

File: components/Main/StyledExercises.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  li {
+    margin: 20px;
+    padding: 10px;
+    margin-bottom: 0;
```
Comment (ufelgen) [inline components/Main/StyledExercises.js]: you can write margin on all 4 sides in one line starting clock-wise from the top: 20px, 20px, 0, 20px (if you only specify 2 values it goes vertically - horizontally)

File: components/Main/StyledExercises.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    text-align: center;
+    box-shadow: 1px 1px 3px 3px #cad2c5;
+  }
+  h2 {
+    font-family: Arial, Helvetica, sans-serif;
+    display: flex;
+    justify-content: center;
+    padding: 5px;
+    margin-top: 20px;
+    margin-bottom: 0;
```
Comment (ufelgen) [inline components/Main/StyledExercises.js]: not sure if margin-bottom needs to be set to 0 (refer to GlobalStyles.js) but you could alternatively use one line again if you wish

**Review by ufelgen — APPROVED** [review]
looks great to me! I made some minor suggestions to simplify your styling :) also I love your description and commit history, very helpful!

**Review by MerleMak — APPROVED** [review]
Nothing to add!! :) (see Ullis remarks)
Great job!

File: components/Main/StyledExercises.js (hunk 2 above)
Comment (esrefbozkurt) [reply components/Main/StyledExercises.js]: after moving the margin and the padding from the body to the universalselector inside Globalstyles, there weren't any default margins left. Thank you!!

**Review by marvinRoellecke — APPROVED** [review]
approved.
just a suggestion you could use responsive unities like rem or em instead of px

**Pull request merged**